media/mathplayground/global.css

```diff
@@ -11,6 +11,15 @@ a:visited {
     color: rgb(209, 76, 242);
 }
 
+a.titlefont {
+    font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
+    color: white;
+}
+a.titlefont:hover, a.titlefont:focus {
+    color: #2196f3;
+    text-decoration: none;
```

Comment:
I think we can leave the link underlined, as it's standard browser styles. This stuff will really be handled by Marc though.. I guess it's pretty subjective. My inclination is always to change the browser defaults as little as possible.

Going forward, I'm sure we'll have an actual logo or something here.